bin/check-log.rb

```diff
@@ -128,6 +128,11 @@ class CheckLog < Sensu::Plugin::Check::CLI
          proc: proc(&:to_i),
          default: 250
 
+  option :return_error_limit,
+         description: 'Max number of returned matched lines\log entries',
```

Comment:
if you want to call out log entries then put it on parenthesis they are not equivalent. Not all logs always are single lines.